wgsl_to_wgpu/src/lib.rs

```diff
@@ -66,6 +66,9 @@ pub enum MatrixVectorTypes {
     /// `glam` types like `glam::Vec4` or `glam::Mat4`.
     /// Types not representable by `glam` like `mat2x3<f32>` will use the output from [MatrixVectorTypes::Rust].
     Glam,
+
+    /// `nalgebra` types like `nalgebra::Vec4` or `glam::Matrix2x3`.
```

Comment:
It doesn't look like this comment matches the nalgebra types from the test case.

Reply:
Heh, you got me :D